build.xml

```diff
@@ -1,38 +1,148 @@
 <?xml version="1.0"?>
```

Comment:
I have never used the `build.xml`, so I'll assume that you know what you're doing and that it works.

Reply:
It works, requires ant 1.9, rather than 1.10 and ant-contrib jar added to the ant lib folder. (I used ant 1.9.9 and ant-contrib 0.6). Line to run is:

ant all -Dversion.num=[versionNumber]

This creates the zipped d4m_api folder.